src/drivers/mct/cime_config/buildexe

```diff
@@ -49,7 +49,7 @@ def _main_func():
     cmd = "%s exec_se -j %d EXEC_SE=%s MODEL=%s LIBROOT=%s -f %s "\
         % (gmake, gmake_j, exename, "driver", libroot, makefile)
 
-    rc, out, err = run_cmd(cmd)
+    rc, out, err = run_cmd(cmd, combine_output=True)
```

Comment:
you can remove err here (replace with "_") since it will never contain anything due to combined output.